spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/cassandra/CassandraDataAutoConfiguration.java

```diff
@@ -80,10 +113,41 @@ public CassandraSessionFactoryBean session(CassandraConverter converter)
 		session.setCluster(this.cluster);
 		session.setConverter(converter);
 		session.setKeyspaceName(this.properties.getKeyspaceName());
-		session.setSchemaAction(SchemaAction.NONE);
+		session.setSchemaAction(SchemaAction.valueOf(this.properties.getSchemaAction()));
```

Comment:
I understood from your [comment on PR 4889](https://github.com/spring-projects/spring-boot/pull/4889#discussion_r48940403) that `CassandraProperties` would not work if Spring Data Cassandra is not on the class path.